### ISSUE TITLE
Positional issues multiple-mesh DFFs

### ISSUE THREAD

**Issue by swift502** [Zarig/DragonFF#13]
Hi,

While importing the map I noticed some of the smaller props are misplaced.

![image](https://user-images.githubusercontent.com/24359130/59453278-d9b9d900-8e0f-11e9-97ac-fec56f4d0ec6.png)
![image](https://user-images.githubusercontent.com/24359130/59453280-dc1c3300-8e0f-11e9-8c3e-77227920c6c1.png)

Turns out some of the barrier models have very strange positions. They should be at 0, 0, 0, but instead they're something else.

![image](https://user-images.githubusercontent.com/24359130/59453641-9d3aad00-8e10-11e9-851a-42d60f3cf1c8.png)

Here's the models in question.
[models.zip](https://github.com/Zarig/DragonFF/files/3287335/models.zip)
[models.tar.gz](https://github.com/Zarig/DragonFF/files/3287336/models.tar.gz)

Any idea what this could be?
Thanks.

**Comment by Zarig** [comment]
Hey,

I'm not sure if the scripts are at fault here. It appears that they are defined to not be at 0,0,0 in the DFF file. Only the first dummy (Group01) is located at (0,0,0)

Thanks.

**Comment by swift502** [comment]
Wow, ok.
I found more. I did export the models with IMG Tool, but I don't think that could've done this. This is really strange.

![image](https://user-images.githubusercontent.com/24359130/59459703-b39b3580-8e1d-11e9-8880-a67f53371f84.png)

**Comment by swift502** [comment]
Yeah I tried it with a different dff importer and it looks the same. It's not an issue with the importer. But I have no idea what it could be. And it is kind of a big deal with the map importer.
